### PR TITLE
130 add remaining indexeddb tests

### PR DESCRIPTION
**Pull request by Valentine1898**

### CODE REVIEW

File: packages/storage/src/indexed-db/indexed-db.test.ts
Comment (grod220) [inline packages/storage/src/indexed-db/indexed-db.test.ts:1]: This file is quite large. Can you either move the test data to another file, or split up the file into multiple tests with test data next to it.

**Review by grod220 — COMMENTED** [review]
I don't think this is every table right? We need tests for:

LAST_BLOCK_SYNCED
TREE_LAST_POSITION
TREE_LAST_FORGOTTEN
TREE_HASHES
TREE_COMMITMENTS
FMD_PARAMETERS
TRANSACTIONS
ASSETS
SPENDABLE_NOTES
NOTES
SWAPS

**Comment by Valentine1898** [comment]
Yes, I've covered all functions

**Comment by grod220** [comment]
Ah ok. In that case, address the comment on the test file length and we'll be good

**Review by grod220 — APPROVED** [review]

**Pull request merged**